### DISCUSSION TITLE
Contributing documentation - running locally out-of-date

### DISCUSSION THREAD

**Discussion by noahjahn** [directus/directus#17736] [Archive]
I recently wanted to contribute to the Directus project but ran into quite a few dead-ends when following the Contributing section in the documentation for Directus.

Specifically on the [Running Locally](https://docs.directus.io/contributing/running-locally.html) page:

- Step 7 appears to have a broken reference to the `directus/docs` repo. Where do we go to update documentation?
- Step 8 appears to document an old way of running tests, the `tests` directory doesn't exist anymore. I stepped back through version history and found tests were changed to `tests-blackbox`. I followed the steps in [.github/actions/prepare/action.yml](https://github.com/directus/directus/blob/228a5ca15f981f01c7072b641d53e1edf562629a/.github/actions/prepare/action.yml) and [.github/workflows/blackbox-main.yml](https://github.com/directus/directus/blob/228a5ca15f981f01c7072b641d53e1edf562629a/.github/workflows/blackbox-main.yml) to be able to run the tests locally

**Comment by abdonrd** [comment]
Looks like the Directus team change the docs repository to private:
https://discord.com/channels/725371605378924594/741317677397704757/1082414711464792128

cc @phazonoverload
